WordPress/src/main/java/org/wordpress/android/ui/photopicker/PhotoPickerFragment.kt

```diff
@@ -84,7 +89,7 @@ class PhotoPickerFragment : Fragment(R.layout.photo_picker_fragment) {
         viewModel = ViewModelProvider(this, viewModelFactory).get(PhotoPickerViewModel::class.java)
     }
 
-    @Suppress("DEPRECATION", "LongMethod")
+    @Suppress("DEPRECATION")
```

Comment:
Warning (⚠️): You will notice that this `DEPRECATION` suppression in no longer necessary for this method, instead and after you resolve the above import comment of mine, you will notice that you now can't build and need to add this on top of the `observeOnPermissionsRequested()` and `observeUIState()` methods, as this is where that belongs now, due to the `PhotoPickerViewModel` being deprecated.

Reply:
i had ran only the gradle commands and moved on, my bad